pyaedt/modeler/cad/Primitives2D.py

```diff
@@ -288,7 +288,8 @@ def create_region(self, pad_percent=300, is_percentage=True):
         ----------
         pad_percent : float, str, list of floats or list of str, optional
             Same padding is applied if not a list. The default is ``300``.
-            If a list of floats or str, interpret as adding for ``["+X", "+Y", "-X", "-Y"]``.
+            If a list of floats or str, interpret as adding ``["+X", "+Y", "-X", "-Y"]`` for XY geometry mode,
```

Comment:
```suggestion
            If a list of floats or strings, interpret as adding ``["+X", "+Y", "-X", "-Y"]`` for XY geometry mode,
```
Avoid abbreviations in descriptions.